cmake configure fails since commit 39d6a8b

<!--- Please fill out this template to the best of your ability. You can always edit this issue once you have created it. -->
<!--- Read the following link before you create a new problem report: https://kodi.wiki/view/HOW-TO:Submit_a_bug_report  -->
## Bug report
### Describe the bug
Here is a clear and concise description of what the problem is:
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
<!--- A bug report that is not clear will be closed -->
<!--- Put your text below this line -->
Since commit 39d6a8bdf5b59ac240a9df3be495d976645a7c96, cmake configure fails on Debian Sid.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:
<!--- Tell us what should happen -->
<!--- Put your text below this line -->
cmake configure should run without error.

## Actual Behavior
<!--- Tell us what happens instead -->
<!--- Put your text below this line -->
cmake configure fails with the following error:
```
CMake Error at cmake/modules/FindFFMPEG.cmake:463 (set_target_properties):
  set_target_properties Can not find target to add properties to:
  ffmpeg::libpostproc
Call Stack (most recent call first):
  cmake/modules/FindFFMPEG.cmake:470 (ffmpeg_create_target)
  cmake/scripts/common/Macros.cmake:383 (find_package)
  cmake/scripts/common/Macros.cmake:395 (find_package_with_ver)
  CMakeLists.txt:299 (core_require_dep)
```

## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->
<!--- Put your text below this line -->
Reverting the commit fixes allow cmake configure to complete successfully, but this is not a valid solution.

FFmpeg removed libpostproc in version 8.0. As a result, Debian no longer packages libpostproc either.

### To Reproduce
Steps to reproduce the behavior:
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
<!--- Put your text below this line -->
1. Debian Sid
2. Install all required build Kodi dependencies.
3. `cmake ../kodi -DCMAKE_INSTALL_PREFIX=/usr -DCORE_PLATFORM_NAME=x11 -DAPP_RENDER_SYSTEM=gl`

### Debuglog
<!--- Put your text below this line -->
<!--- A debuglog is always mandatory when creating an issue. Provide one! -->
The CMakeConfigureLog can be found here: [https://paste.kodi.tv/oneguyeget](https://paste.kodi.tv/oneguyeget)

### Screenshots 
Here are some links or screenshots to help explain the problem:
<!--- Put your text below this line -->
N/A

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:
<!--- How has this bug affected you? What were you trying to accomplish? -->
<!--- Put your text below this line -->
N/A

### Your Environment
Used Operating system:
<!--- Include as many relevant details about the environment you experienced the bug in -->
<!--- Put your text below this line. Checkboxes can easily be ticked once issue is created -->
 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [x] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version: 22.0a3-Piers-271-gc35e1ca4bb (git HEAD)

<!--- End of this issue -->
*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

--- a/cmake/modules/FindFFMPEG.cmake
+++ b/cmake/modules/FindFFMPEG.cmake
@@ -457,14 +457,14 @@ else()
                                                    IMPORTED_LOCATION "${FFMPEG_${libname_UPPER}}"
                                                    INTERFACE_LINK_LIBRARIES "${${libname}_LDFLAGS}")
         endif()
-      endif()
 
-      if(FFMPEG_${libname_UPPER}_INCLUDE_DIRS)
-        set_target_properties(ffmpeg::${libname} PROPERTIES
-                                                 INTERFACE_INCLUDE_DIRECTORIES "${FFMPEG_${libname_UPPER}_INCLUDE_DIRS}")
-      else()
-        set_target_properties(ffmpeg::${libname} PROPERTIES
-                                                 INTERFACE_INCLUDE_DIRECTORIES "${FFMPEG_INCLUDE_DIRS}")
+        if(FFMPEG_${libname_UPPER}_INCLUDE_DIRS)
+          set_target_properties(ffmpeg::${libname} PROPERTIES
+                                                   INTERFACE_INCLUDE_DIRECTORIES "${FFMPEG_${libname_UPPER}_INCLUDE_DIRS}")
+        else()
+          set_target_properties(ffmpeg::${libname} PROPERTIES
+                                                   INTERFACE_INCLUDE_DIRECTORIES "${FFMPEG_INCLUDE_DIRS}")
+        endif()
       endif()
     endmacro()
 

